client-misk/src/main/kotlin/app/cash/backfila/client/misk/BackfillConfig.kt

```diff
@@ -3,6 +3,15 @@ package app.cash.backfila.client.misk
 import okio.ByteString
 
 data class BackfillConfig(
+  /**
```

Comment:
nit: Generally preferred to have KDocs be on the class instead of on the member/val